db/seeds.rb

```diff
@@ -5,9 +5,27 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+Category.create(name: "drama")
```

Comment:
Thanks for the trick. A small change in the code that can be a big time saver. I didn´t use the bang methods often, from now I will.